### PR TITLE
Add loginWithToken docs

### PR DESCRIPTION
**Pull request by freiksenet**

### CODE REVIEW

**Comment by freiksenet** [comment]
@fson ping

File: README.md [hunk 1]
```diff
@@ -68,6 +68,37 @@ Token is stored inside the instance and inside browser `localStorage`.
...
+    return;
+  }
+
+  const reindexResponse = await reindex.loginWithToken(
+    'auth0',
+    auth0Result[1],
+  );
+  for (const error of reindexResponse.errors || []) {
+    console.log(error);
+  }
```
Comment (fson) [inline README.md]: Please copy the fixed example code from docs: https://www.reindex.io/docs/security/authentication/#login-with-auth0-lock

File: README.md [hunk 2]
```diff
@@ -68,6 +68,37 @@ Token is stored inside the instance and inside browser `localStorage`.
 
 Emits `login` and `tokenChange` events.
 
+##### `.loginWithToken(providerName: String, loginToken: String)`
+
+Attempt to login to Reindex with a login token provided by some provider.
```
Comment (fson) [inline README.md]: s/a token provided by some provider/an Auth0 `id_token`/

**Comment by freiksenet** [comment]
@fson all done.

**Comment by fson** [comment]
Merged with minor fixes.

**Pull request merged**